_topic_maps/_topic_map_rosa.yml

```diff
@@ -306,6 +306,19 @@ Topics:
   - Name: AWS Elastic File Service CSI Driver Operator
     File: rosa-persistent-storage-aws-efs-csi
 ---
+Name: Registry
+Dir: registry
+Distros: openshift-rosa
+Topics:
+- Name: Registry overview
+  File: index
+- Name: Image Registry Operator in OpenShift Container Platform
```

Comment:
@bhardesty Does it need to update OpenShift Container Platform to "Openshift ROSA" ?

Reply:
Fixed.